packages/expo/src/winter/ImportMetaRegistry.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import { getBundleUrl } from '../utils/getBundleUrl';
+
+/**
+ * Registry to handle import.meta functionality for React Native environment
+ * Similar to how it works in the web, but adapted for the RN context
+ * https://github.com/wintercg/import-meta-registry
+ */
+export const ImportMetaRegistry = {
+  get url() {
+    return getBundleUrl();
+  },
+  get env() {
+    return process.env;
+  },
```

Comment:
Let's remove this for now since it would require more rules around client vs server.